src/main/java/no/nav/familie/inntektsmelding/forespørsel/modell/OmsorgspengerForespørselEntitet.java

```diff
@@ -0,0 +1,108 @@
+package no.nav.familie.inntektsmelding.forespørsel.modell;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+
+@SequenceGenerator(name = "GLOBAL_PK_SEQ_GENERATOR", sequenceName = "SEQ_GLOBAL_PK")
+@Entity(name = "OmsorgspengerForespørselEntitet")
+@Table(name = "OMSORGSPENGER_FORESPOERSEL")
+public class OmsorgspengerForespørselEntitet {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "GLOBAL_PK_SEQ_GENERATOR")
+    private Long id;
+
+    @OneToOne(optional = false)
+    @JoinColumn(name = "FORESPOERSEL_ID", nullable = false, updatable = false)
+    private ForespørselEntitet forespørsel;
+
+    @Column(name = "BEGRUNNELSE_FOR_SOEKNAD", nullable = false)
+    private String begrunnelseForSøknad;
+
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "omsorgspengerForespørsel")
+    private List<FraværsPeriodeForespørselEntitet> fraværsPerioder = new ArrayList<>();
+
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "omsorgspengerForespørsel")
+    private List<DelvisFraværsDagForespørselEntitet> delvisFraværsDager = new ArrayList<>();
+
+    public OmsorgspengerForespørselEntitet() {
```

Comment:
Disse default konstruktørene for hibernate trenger ikke være public, og bør heller ikke være det for å hindre misbruk. Gjelder alle de nye entitetene her.